### DISCUSSION TITLE
How to use the examples

### DISCUSSION THREAD

**Discussion by hqasmei** [clauderic/dnd-kit#1245] [Q&A]
Hi,

I wanted to know if it’s possible to use the examples from https://master--5fc05e08a4a65d0021ae0bf2.chromatic.com/. I’m trying to use the Sortable/Grid/Drag Handle. When I press show I see what’s in the screenshot. Not sure how to use that code if I have to download something, or it’s just an example, and I have to create my own. Would appreciate the help. Thank you.
![IMG_1544](https://github.com/clauderic/dnd-kit/assets/39573679/1f10cf07-52c7-4277-b66d-7857fac605a5)

**Comment by akhalid-dev** [comment]
@hqasmei I believe this is the code for the examples:
https://github.com/clauderic/dnd-kit/tree/master/stories/3%20-%20Examples
